splunk/com/splunk/HttpCookie.java

```diff
@@ -0,0 +1,1163 @@
+/*
```

Comment:
Can you edit the project's readme and include 'in big text' or something like that:
This branch can not be distributed.  Mainly because we're using a modified copy of HttpCookie from Oracle and the license restricts distribution.

@oesolutions, is there a way to disable or limit visibility of this branch to non-members?